docs/admin/authorization.md

```diff
@@ -53,7 +53,7 @@ A request has the following attributes that can be considered for authorization:
   - what resource is being accessed (for resource requests only)
   - what subresource is being accessed (for resource requests only)
   - the namespace of the object being accessed (for namespaced resource requests only)
-  - the API group being accessed (for resource requests only)
+  - the API group being accessed (for resource requests only); the [core API group](../api.md#api-groups) is identified by the empty string
```

Comment:
I was thinking "the emtpy string" is correct in the sense that there is only one empty string.  Every other string is not empty.  This is viewing a string as a sequence of characters, not a particular data structure at a particular address in memory.  I think my choice is proper for a golang context, since that is how go treats strings.  For example, you can find mention of "the empty string" at https://tour.golang.org/basics/12